File: CESMAC 4° periodo/POO/AV1/ESTUDOS/CURSO POO BOSON TREINAMENTOS/Método Construtor __init__/main.py
class Gato:
    def __init__(self,nome,cor,idade):
        self.nome = nome
        self.cor = cor 
        self.idade = idade


if __name__ == "__main__":
    gato = Gato("Mingau", "Cinza", 2)
    print(f"Nome do gato: {gato.nome}")
    print(f"Idade do gato em anos humanos: {gato.calcular_idade_em_anos_humanos()}")

File: CESMAC 4° periodo/POO/AV1/ESTUDOS/CURSO POO BOSON TREINAMENTOS/Método Construtor __init__/main.java


public class Main{
   public static class Gato{

        public String nome;
        public String cor;
        public int idade;

       public Gato(String nome,String cor, int idade){
            this.nome = nome
            this.cor = cor
            this.idade = idade
       }
   }
	public static void main(String[] args) {
        Gato gato = new Gato("Mingau", "Cinza", 2);
        System.out.println("Nome do gato: " + gato.nome);
		
	}
}